src/idd/idd.heatmap.js

```diff
@@ -507,7 +774,18 @@ InteractiveDataDisplay.Heatmap = function (div, master) {
         },
         configurable: false
     });
-
+    Object.defineProperty(this, "interval", {
```

Comment:
@ElenaPochernina  It is better to remove this property; it has some subtle issues. The only way to set the interval is to call 'draw'.